### PR TITLE
Travis CI: Run tests on Python 3.11 (alpha)

### PR DESCRIPTION
**Pull request by cclauss**

### CODE REVIEW

**Review by ilevkivskyi — APPROVED** [review]

**Comment by cclauss** [comment]
Probably more like [Python 3.11.0-beta.3](https://github.com/actions/python-versions/releases) these daze.

File: .travis.yml [hunk 1]
```diff
@@ -20,8 +23,12 @@ jobs:
       python: 3.9
       script: export PYTHONPATH="src"; python -m unittest src/test_com2ann.py
 
-    - name: "Run tests on Python 3.10 (beta)"
-      python: 3.10-dev
+    - name: "Run tests on Python 3.10.2"
+      python: 3.10.2
```
Comment (ilevkivskyi) [inline .travis.yml:27]: Actually I think it may be better to try the latest 3.10
```suggestion
      python: 3.10
```

Comment (cclauss) [reply .travis.yml]: "3.10" MUST be quoted in YAML... https://dev.to/hugovk/the-python-3-1-problem-85g

**Comment by ilevkivskyi** [comment]
> Probably more like [Python 3.11.0-beta.3](https://github.com/actions/python-versions/releases) these daze.

If tests will pass I think plain `-dev` is best.

File: .travis.yml [hunk 2]
```diff
@@ -20,8 +23,12 @@ jobs:
       python: 3.9
       script: export PYTHONPATH="src"; python -m unittest src/test_com2ann.py
 
-    - name: "Run tests on Python 3.10 (beta)"
-      python: 3.10-dev
+    - name: "Run tests on Python 3.10.2"
```
Comment (ilevkivskyi) [inline .travis.yml:26]: Oh, forgot the action title.
```suggestion
    - name: "Run tests on Python 3.10"
```

**Comment by cclauss** [comment]
Do you pay a subscription for Travis CI?

**Comment by ilevkivskyi** [comment]
> Do you pay a subscription for Travis CI?

No, I think it is free (for small number of jobs).

**Comment by cclauss** [comment]
Still 3.11 alpha, not yet beta.  https://app.travis-ci.com/github/ilevkivskyi/com2ann/jobs/572204239#L195

**Pull request merged**